### PR TITLE
PCX-1042 SSL-Restore missing content

### PR DESCRIPTION
**Pull request by aaron-ruby-df**
## User story

Port the content for PCX-877 to the new cloudflare-docs Github repository so that customers have access to published content that was omitted when the SSL docs were migrated.

### CODE REVIEW

File: products/ssl/src/content/ssl-for-saas/status-codes/custom-csrs.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 order: 2
```
Comment (adamschwartz) [inline products/ssl/src/content/ssl-for-saas/status-codes/custom-csrs.md:3]: I would add a shorter frontmatter for the sidebar:

```txt
title: Custom CSRs
```

**Review by adamschwartz — CHANGES_REQUESTED** [review]
One small change.

File: products/ssl/src/content/ssl-for-saas/status-codes/custom-csrs.md (hunk 1 above)
Comment (aaron-ruby-df) [reply products/ssl/src/content/ssl-for-saas/status-codes/custom-csrs.md:3]: Done.

File: products/ssl/src/content/ssl-for-saas/issuing-certificates.md [hunk 2]
```diff
@@ -46,9 +46,8 @@ $ curl -XGET "https://api.cloudflare.com/client/v4/zones/{zone_id}/custom_hostna
 
 ## Viewing the newly issued certificate
 
-You can view the certificate that was deployed to Cloudflare’s edge using `openssl` or your browser. The command below can be used in advance of your customer pointing the `app.example.com` hostname to the edge ([provided validation was completed](/ssl-for-saas/validation-methods)).
+You can view the certificate that was deployed to Cloudflare’s edge using `openssl` or your browser. The command below can be used in advance of your customer pointing the `app.example.com` hostname to the edge ([provided validation was completed](/ssl-for-saas/certificate-validation-methods)).
 
-```txt
-$ openssl s_client -servername app.example.com -connect $CNAME_TARGET:443 </dev/null 2>/dev/null | openssl x509 -noout -text | grep app.example.com
+```sh
```
Comment (adamschwartz) [inline products/ssl/src/content/ssl-for-saas/issuing-certificates.md:51]: The`sh` code block line should start with `$`. See:
https://developers.cloudflare.com/docs-engine/reference/markdown#terminals

**Review by adamschwartz — CHANGES_REQUESTED** [review]

File: products/ssl/src/content/ssl-for-saas/issuing-certificates.md (hunk 2 above)
Comment (adamschwartz) [reply products/ssl/src/content/ssl-for-saas/issuing-certificates.md:51]: (The line below)

**Review by adamschwartz — APPROVED** [review]

**Pull request merged**